README.rst

```diff
@@ -47,13 +47,13 @@ The short version:
 
 .. code-block:: shell
 
-    $ virtualenv -p python3.6 ~/libertem-venv/
+    $ virtualenv -p python3 ~/libertem-venv/
     $ source ~/libertem-venv/bin/activate
     (libertem) $ pip install libertem[torch]
```

Comment:
This line should also have quotes, like the corresponding line in the documentation.

Reply:
Oops! forgot that one, Fixed :smile: